### ISSUE TITLE
Post Mortem service status type

### ISSUE THREAD

**Issue by SuperSajuuk** [adamcooke/staytus#343]
In major outage situations, its often common place to write a "post mortem" if an issue has occurred that was significantly impactful to services. It'd be useful to have a "Post Mortem" status that can be used when a ticket has been moved to the Resolved state, allowing the documentation of how the issue occurred, the time line and what has been done to remediate the issue to prevent it occurring again in the future.

**Comment by misterjaytee** [comment]
You could add this information as an update to the issue itself after you have resolved the issue. It would then be available in the history of that issue.

**Comment by SuperSajuuk** [comment]
> You could add this information as an update to the issue itself after you have resolved the issue. It would then be available in the history of that issue.
Indeed, but that would come under a "Resolved" status. Additionally, status posts don't respect formatting like newlines, so its just one lengthy run-on paragraph which isn't very easy to read if you are doing a PM :P

On something like statuspage.io, there is a type called "postmortem" which is basically a reworded "resolved" but instead of it being small updates, its a multi-paragraph "opening statement" that effectively explains the reasons behind the status post. I would use statuspage.io but it has severe limitations and isn't suitable for my use case (staytus covers what i need fairly well)

An example of a statuspage post with a PM on it can be found here if it helps to explain what I am referring to: https://discordstatus.com/incidents/mpzkz7m4fzpt

You can see that the PM status is effectively "separate" from the regular updates, allowing for more free-form "explanation". Its a minor thing and unlikely to have any impact for me if such a thing wasn't added, but it'd nice to have additional options :)
